src/array.rs

```diff
@@ -447,6 +473,9 @@ impl<T: Element, D: Dimension> PyArray<T, D> {
     /// If `is_fortran` is true, then
     /// a fortran order array is created, otherwise a C-order array is created.
     ///
+    /// For elements with `DATA_TYPE == DataType::Object`, this will fill the array
+    /// valid pointers to objects of type `<class 'int'>` with value zero.
```

Comment:
```suggestion
    /// For elements with `DATA_TYPE == DataType::Object`, this will fill the array
    /// with valid pointers to zero-valued Python integer objects.
```

Reply:
:+1: